msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -9327,6 +9327,11 @@ public void setRankBeanList(List ranklist) {
 		this.rankBeanList.addAll(alist);
 	}
 
+	public boolean isRanksEnabled()
+	{
+		return ServerConfigurationService.getBoolean("msgcntr.forums.ranks.enable", true);
```

Comment:
I'm just wondering how often this would get called. Maybe it would be better to store the result of `ServerConfigurationService.getBoolean("msgcntr.forums.ranks.enable", true);` in a class variable and return that, rather than hitting SCS every time.